# --- SDE-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
#
# Filename: package/.../lua-base64/lua-base64-make-install.patch
# Copyright (C) 2010 The OpenSDE Project
#
# More information can be found in the files COPYING and README.
#
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- SDE-COPYRIGHT-NOTE-END ---

Description: Add an 'install' make target and set reasonable default directories

--- base64/Makefile.orig	2010-10-21 13:16:32.650692078 +0200
+++ base64/Makefile	2010-10-21 13:44:07.890695712 +0200
@@ -1,16 +1,12 @@
 # makefile for base64 library for Lua
 
-# change these to reflect your Lua installation
-LUA= /tmp/lhf/lua-5.1.4
-LUAINC= $(LUA)/src
-LUALIB= $(LUA)/src
-LUABIN= $(LUA)/src
-
 # these will probably work if Lua has been installed globally
-#LUA= /usr/local
-#LUAINC= $(LUA)/include
-#LUALIB= $(LUA)/lib
-#LUABIN= $(LUA)/bin
+LUA= /usr/local
+LUAINC= $(LUA)/include
+LUALIB= $(LUA)/lib
+LUABIN= $(LUA)/bin
+
+LUACMOD= $(LUALIB)/lua/5.1
 
 # probably no need to change anything below here
 CC= gcc
@@ -45,6 +41,10 @@ doc:
 	@echo "$(MYNAME) library:"
 	@fgrep '/**' $(MYLIB).c | cut -f2 -d/ | tr -d '*' | sort | column
 
+install: all
+	mkdir -p $(LUACMOD)
+	cp -v $T $(LUACMOD)
+
 # distribution
 
 FTP= www:www/ftp/lua/5.1
